javascript-practice/src/views/components/product-item.html

```diff
@@ -0,0 +1,27 @@
+<div class="product-item">
+  <figure class="product-thumbnail">
+    <a href="javascript:void(0)">
+      <img src="./assets/images/apple-airpods.png" alt="Apple Airpods">
+    </a>
+  </figure>
+  <div class="product-details">
+    <a href="javascript:void(0)">
+      <h2 class="product-name">
+        Apple Airpods
+      </h2>
+    </a>
+    <ul class="product-option-colors">
+      <li class="product-option-color">
+        <button class="btn btn-option-color btn-option-light">
```

Comment:
Should use <span> instead of <button> here.

Reply:
Could you clarify your comment?

Reply:
I mean you don't use a button tag here. Should use span or other tag. Currently, I see you updated this.